db/api/v1/routes/validate.js

```diff
@@ -104,15 +120,102 @@ const parseId = (idStr) => {
         : ParseResult.success(id);
 };
 
+/**
+ *  Validates if it is a string and within min/max length
+ * 
+ * @param {*} name - the variable to parse
+ * @param {*} [minLength=0] - the minimum length to pass.
+ * @param {*} [maxLength=Number.MAX_SAFE_INTEGER] - the maximum length to pass
```

Comment:
The type of the minLength and maxLength can be {number}. Because you set a default value, the case where the argument of this parameter is undefined is covered.